alerting/provider/mattermost/mattermost.go

```diff
@@ -70,6 +70,7 @@ func (provider *AlertProvider) Send(ep *endpoint.Endpoint, alert *alert.Alert, r
 }
 
 type Body struct {
+	Channel     string       `json:"channel,omitempty"` // Optional channel override
```

Comment:
This only adds the channel parameter to the body, it doesn't actually expose a channel configuration parameter through `alerting.mattermost.channel` as your change in the README.md would indicate.

You also have to modify https://github.com/TwiN/gatus/blob/93d8a813d2ffb7f43cf9ae5c6b39d5e0ff592cd6/alerting/provider/mattermost/mattermost.go#L16

and you would have to set https://github.com/TwiN/gatus/blob/93d8a813d2ffb7f43cf9ae5c6b39d5e0ff592cd6/alerting/provider/mattermost/mattermost.go#L121-L122
to
```
 body := Body{ 
 	Channel:  provider.Channel, 
```